conjur_api/models/hostfactory/create_host_data.py

```diff
@@ -19,17 +19,19 @@ class CreateHostData:
     def __init__(self,
                  # using id shadows the internal id
                  host_id: str = "",
-                 token: str = ""):
+                 token: str = "",
+                 annotations: dict = None):
```

Comment:
```suggestion
                 annotations: dict = {}):
```
Would this work? And then you won't need the line above will be:
```python
self.annotations = annotations
```

Reply:
That's what I did at first but it fails the lint. Apparently, if the default parameter changes in the function then the change will apply for the next time someone calls this function